LuaUI/Widgets/gui_chili_proconsole2.lua

```diff
@@ -10,7 +10,7 @@ function widget:GetInfo()
 		license      = "GNU GPL, v2 or later",
 		layer        = 50,
 		experimental = false,
-		enabled      = true,
+		enabled      = false,

```

Comment:
TODO: Check the new console

Reply:
What is definition of done of this TODO? 😈 Maybe I can assist somehow by answering some questions.

Reply:
Just take a look... I even focused on that last time I played your UI